trace/cloud-trace-demo-app/app/demo-service-c.yaml.template

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
```

Comment:
is this template needed if the service c yaml  isn't included?

Reply:
This template is needed to generate service c yaml file since the IP host is going to be dynamic.